concepticondata/conceptlists.tsv

```diff
@@ -193,3 +194,5 @@ Stokhof-1975-117	Stokhof	1975		117	basic,areal	English	New Guinea	A "representat
 Clouse-1993-71	Clouse	1993		71	basic,areal	English	New Guinea	A wordlist designed to elucidate the relationships between Kirikiri and other Lakes Plains languages	Clouse1993	Clouse1993		29-30	
 Smith-2015-97	Smith	2015		91	basic,areal	English	Borneo	A 97 item wordlist designed to demonstrate the differences and relationships between the Kenyah and Kayan languages of Borneo	Smith2015	Smith2015		351-355	
 Laycock-1968-50	Laycock	1968		50	basic,areal	English	New Guinea	A 50 item wordlist	Laycock1968	Laycock1968		51-363	
+Voorhoeve-1971-125	Voorhoeve	1971		125	basic,areal	English	New Guinea	A 125 item wordlist survey of West Irian	Voorhoeve1971	Voorhoeve1971		98-114	
+Larson-1972-68	Larson & Larson	1972		68	basic,areal	English	New Guinea	A 68 item wordlist based on [Swadesh 100](:ref:Swadesh-1955-100).	Larson1972	Larson1972		90-92	
```

Comment:
sorry, I mean "lastname, firstname" AND "lastname2, firstname2", so the same regulations as in bibtex, not just the lastname, in the author column.

E.g.:

```tsv
Smith-2015-97	Smith	2015		91	basic,areal	English	Borneo	A 97 item wordlist designed to demonstrate the differences and relationships between the Kenyah and Kayan languages of Borneo	Smith2015	Smith2015		351-355	
```

change to:

```tsv
Smith-2015-97	Smith, Alexander D.	2015		91	basic,areal	English	Borneo	A 97 item wordlist designed to demonstrate the differences and relationships between the Kenyah and Kayan languages of Borneo	Smith2015	Smith2015		351-355	
```

So use AND as delimiter if there is more than one (first last or last, first, such as in bibtex).